templates/slim/block_image.html.slim

```diff
@@ -1,10 +1,11 @@
-.imageblock(id=@id class=[@style,role]
-    style=[("text-align: #{attr :align}" if attr? :align),("float: #{attr :float}" if attr? :float)].compact.join('; '))
-  .content
-    - if attr? :link
-      a.image href=(attr :link)
+- unless attr(1) == 'background'
```

Comment:
this patch looks larger than it should due to diff being confused. all I did is inject this unless statement.